gunicorn/workers/geventlet.py

```diff
@@ -14,9 +14,8 @@
     raise RuntimeError("You need eventlet installed to use this worker.")
 
 # validate the eventlet version
-if eventlet.version_info < (0, 9, 7):
-    raise RuntimeError("You need eventlet >= 0.9.7")
-
+if eventlet.version_info < (0, 19):
```

Comment:
The [NEWS] file for Eventlet mentions a lot of important Python 3 fixes for 0.18, but the first two patch releases in that series have serious issues. I chose 0.19 to be safe because it includes a number of other fixes that look good and that version is almost three years old now, anyway, so it's not an aggressive upgrade.